core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -281,9 +281,19 @@ private[spark] object TestUtils {
     attempt.isSuccess && attempt.get == 0
   }
 
-  def isPythonVersionAtLeast38(): Boolean = {
+  // SPARK-40053: This string needs to be updated when the
+  // minimum python supported version changes.
+  val minimumPythonSupportedVersion: String = "3.7.0"
+
+  def isPythonVersionAvailable: Boolean = {
```

Comment:
https://github.com/apache/spark/blob/cf1a80eeae8bf815270fb39568b1846c2bd8d437/sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala#L108

Should we replace `TestUtils.testCommandAvailable(pythonExec)` with `TestUtils.isPythonVersionAvailable` ?